### PR TITLE
JR/SC-1969/doc client rename components and routes

### PR DESCRIPTION
**Pull request by enrjp**
This PR most importantly updates the routes as requested in the Shortcut Ticket https://app.shortcut.com/ameelio/story/1969/doc-client-rename-components-and-routes.

In updating the routes, I also thought it made sense to update accompanying code like variable or parameter names since it felt awkward to just change path names but leave everything else as "Contact" but just the paths updated to visitor. But if anything feels _too_ invasive please let me know and I'm happy to revert 👍🏻 .

### CODE REVIEW

File: src/api/graphql.ts [hunk 1]
```diff
@@ -315,6 +315,17 @@ export type CreateGroupResponse = {
   group: Group;
```
Comment (enrjp) [inline src/api/graphql.ts:315]: This is just from running update graphql

File: src/components/Profile/MeetingsTable.tsx [hunk 2]
```diff
@@ -51,7 +51,7 @@ export default function MeetingsTable({ meetings }: Props) {
         )}
       />
       <Table.Column
-        title="Contact(s)"
+        title="Visitor(s)"
         dataIndex="visitors"
```
Comment (enrjp) [inline src/components/Profile/MeetingsTable.tsx:55]: For example re: invasiveness, maybe we still want the label to say "Contact(s)" - not sure...

File: src/pages/CallRequestsPage/CallRequests.tsx [hunk 3]
```diff
@@ -66,7 +66,7 @@ export const CallRequests = ({
         )}
       />
       <Table.Column
-        title={isProviders ? "Professional" : "Contact"}
+        title={isProviders ? "Professional" : "Visitor"}
```
Comment (enrjp) [inline src/pages/CallRequestsPage/CallRequests.tsx:69]: Same here

File: src/components/Profile/MeetingsTable.tsx (hunk 2 above)
Comment (wittejm) [reply src/components/Profile/MeetingsTable.tsx:55]: Yeah this task is a little awkward. We've dug in pretty hard on `Visitor` as a name in the data model for any outside user, but it doesn't make as much sense for user-facing language so maybe keeping this as "Contact" is best.

File: src/pages/CallRequestsPage/CallRequests.tsx (hunk 3 above)
Comment (wittejm) [reply src/pages/CallRequestsPage/CallRequests.tsx:69]: ^^

**Review by wittejm — APPROVED** [review]
This PR feels so good to read. I had forgotten how irked I felt about these names.

**Review by cainlevy — APPROVED** [review]

File: src/pages/routes.tsx [hunk 4]
```diff
@@ -39,9 +39,9 @@ type Domain =
   | "in-person-visits"
   | "staff"
   | `staff/${string}`
-  | "members"
-  | `members/${string}`
-  | `contacts/${string}`
```
[the remark this answers is not in the document]
Comment (enrjp) [reply src/pages/routes.tsx:44]: Sure!

**Pull request merged**